benchmark/Chart.hs

```diff
@@ -423,3 +427,13 @@ main = do
                             (makeGraphs "fold-o-n-heap")
                             "charts/fold-o-n-heap/results.csv"
                             "charts/fold-o-n-heap"
+                UnfoldO1Space -> benchShow opts cfg
```

Comment:
This is all mechanically repeated code, at some point we have to do something about it, maybe use TH or some other way to remove duplication.